apps/api/src/database/repositories/competition-repository.ts

```diff
@@ -1755,6 +1755,59 @@ export const findActiveCompetitionsPastEndDate = createTimedRepositoryFunction(
   "findActiveCompetitionsPastEndDate",
 );
 
+/**
+ * Get portfolio timeline for agents in a competition
+ * @param competitionId Competition ID
+ * @returns Array of portfolio timelines per agent
+ */
+async function getAgentPortfolioTimelineImpl(competitionId: string) {
+  try {
+    const result = await dbRead.execute<{
+      date: string;
+      agent_id: string;
+      agent_name: string;
+      competition_id: string;
+      total_value: number;
+    }>(sql`
+      SELECT DATE(timestamp) AS date, agent_id, name AS agent_name, competition_id, total_value FROM (
```

Comment:
I'm not sure if we want to be using DATE() here.  This removes the time infomation and just returns year-month-day.  If you run this query on the prod data set for the comp last week, you'll get 130 rows, and they look like below:

```
2025-07-23, <agent_id>, Vincent, <comp_id>, 30010.500000000000000
2025-07-24, <agent_id>, Vincent, <comp_id>, 29983.920365335984000
```

This works for the 24 hour comp, but if the comp was 12 hours you wouldn't know which row was older/newer.

Reply:
I see, I guess this endpoint is only for data that is displayed on a 24 basis, so seems ok.